Don't install C libs by default since we don't need to install them for
every python version since they aren't linked with libpython.

--- binwalk-2.0.1/setup.py
+++ binwalk-2.0.1/setup.py
@@ -134,7 +134,7 @@
 
 # The data files to install along with the module
 data_dirs = ["magic", "config", "plugins", "modules", "core"]
-install_data_files = [os.path.join("libs", "*.so")]
+install_data_files = []
 
 for data_dir in data_dirs:
     install_data_files.append("%s%s*" % (data_dir, os.path.sep))
